src/connection-manager/connections.ts

```diff
@@ -120,6 +55,15 @@ async function setHandlers(
       void saveNodeWsUrl(ws.url, true)
       connections.set(ip, ws)
       subscribe(ws)
+
+      // Use LedgerEntry to look for fee data to apply the fee for all validators
+      // in a network that doesn't vote for a new fee. We want to do it only once
+      // per network. This part will check if the node is the initial node that we
```

Comment:
Why do we want to do this only once per network?

Reply:
And why not make that a separate call that happens?

Reply:
Other details for these already added amendments will also be added manually via the `amendments_enabled.json` file (but only ones that had been enabled via EnableAmendment tx in the past, those are added on genesis will have `ledger_index`, `tx_hash` and `date` fields blank